lib/myFTClient.js

```diff
@@ -828,28 +1051,81 @@ function removeGroup(uuid) {
+ * @param {Object} relationshipProperties - properties to add to the 'member' relationship(s)
+ * @return {Promise} response -
+**/
+function addConceptsFollowedByKatUser(userUUIDs, conceptUUIDs, relationshipProperties) {
+  log.debug({operation: `addConceptsFollowedByKatUser`});
+  return _addRemoveKatConceptsFollowed('POST',myftConst.userNodeName,userUUIDs, conceptUUIDs, relationshipProperties);
+}
+
+
+/**
  * Add topics for a group to follow
  * @param {String|Array} groupUUID - uuid of the user
  * @param {String|Array} conceptUUIDs - uuid of the topic, or an array of topic uuids, to follow
  * @param {Object} relationshipProperties - properties to add to the 'member' relationship(s)
  * @return {Promise} response -
 **/
-function addConceptsFollowedByGroup (groupUUID, conceptUUIDs, relationshipProperties) {
-  return _addRemoveConceptsFollowed('POST', myftConst.groupNodeName, groupUUID, conceptUUIDs, relationshipProperties);
+//TODO Depreciate in v2.0.0
+function addConceptsFollowedByGroup(groupUUID, conceptUUIDs, relationshipProperties) {
+  return _addRemoveConceptsFollowed('POST', myftConst.groupNodeName , groupUUID, conceptUUIDs, relationshipProperties);
+
+}
+
+/**
+ * Add topics for a group to follow
+ * @param {String|Array} groupUUID - uuid of the group
+ * @param {String|Array} conceptUUIDs - uuid of the topic, or an array of topic uuids, to follow
+ * @param {Object} relationshipProperties - properties to add to the 'member' relationship(s)
+ * @return {Promise} response -
+**/
+function addConceptsFollowedByKatGroup(groupUUIDs, conceptUUIDs, relationshipProperties) {
+  return _addRemoveKatConceptsFollowed('POST', myftConst.groupNodeName , groupUUIDs, conceptUUIDs, relationshipProperties);
+}
+
+/**
+ * Add topics for group members to follow
+ * @param {String|Array} groupUUID - uuid of the group
+ * @param {String|Array} conceptUUIDs - uuid of the topic, or an array of topic uuids, to follow
+ * @param {Object} relationshipProperties - properties to add to the 'member' relationship(s)
+ * @return {Promise} response -
+**/
+function addConceptsFollowedByKatGroupMembers(groupUUID, conceptUUIDs, relationshipProperties) {
+  return _addRemoveKatConceptsFollowed('POST', myftConst.groupNodeName , groupUUID, conceptUUIDs, relationshipProperties, true);
+}
+
+/**
+ * Add topics for specific group members to follow
+ * @param {String|Array} groupUUID - uuid of the group
+ * @param {String|Array} userUUIDs - uuids of the group users
+ * @param {String|Array} conceptUUIDs - uuid of the topic, or an array of topic uuids, to follow
+ * @param {Object} relationshipProperties - properties to add to the 'member' relationship(s)
+ * @return {Promise} response -
+**/
+function addConceptsFollowedByKatGroupMembSpec(userUUIDs, conceptUUIDs, relationshipProperties, groupUUID) {
+  log.silly({operation: 'acfbkgms', groupUUID, userUUIDs: JSON.stringify(userUUIDs), conceptUUIDs: JSON.stringify(conceptUUIDs), relationshipProperties});
```

Comment:
Do you still want to keep this log in here since the `operation` has a dummy value?